apps/web/pages/api/user/avatar.ts

```diff
@@ -26,6 +28,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       email: user?.email,
       avatar: user?.avatar,
     };
+    linksToThisRoute =
+      identity.avatar === `${CAL_URL}/${username}/avatar.png` ||
+      identity.avatar === `${WEBAPP_URL}/${username}/avatar.png`;
```

Comment:
CAL_URL its WEBAPP_URL at the end, so we should only look for the second statement right?
<img width="679" alt="image" src="https://user-images.githubusercontent.com/4461358/199120713-b86242bc-34c4-4d0d-873c-2585bd5751e3.png">

Reply:
Nope. It is WEBAPP_URL for previews only(which ends in .vercel.app). For the production app it is WEBSITE_URL.

Also, considering that we use WEBAPP_URL and WEBSITE_URL interchangeably many times, it is safer to consider both endpoints